src/variationallaplace/transferfunction.jl

```diff
@@ -223,7 +223,7 @@ function csd_fmri_mtf!(y, x, ω, p, params)
     for i = 1:nr
         for j = i:nr
             Gn[:, i, j] .+= exp(params.lnβ[1])*G
-            Gn[:, j, i] = Gn[:, i, j]
+            Gn[:, j, i] .= Gn[:, i, j]
```

Comment:
this doesn't seem to do anything but I guess it is good practice to assign point-wise?